src/rpcblockchain.cpp

```diff
@@ -367,8 +367,6 @@ UniValue gettxoutsetinfo(const UniValue& params, bool fHelp)
             + HelpExampleRpc("gettxoutsetinfo", "")
         );
 
-    LOCK(cs_main);
-
     UniValue ret(UniValue::VOBJ);
 
     CCoinsStats stats;
```

Comment:
Does ``FlushStateToDisk`` (on the next line) not require the lock? It sounds side-effecty.